### PR TITLE
docs: fixed missing references

### LINKED ISSUE #9: docs: incomplete objects.inv for intersphinx
Published `objects.inv` on Read the Docs does not contain all references used by Goblin.

### PR DESCRIPTION
**Pull request by jirikuncar**
* Allows building documentation in strict mode with:
  `sphinx-build -qnNW docs docs/_build/html`  (closes #9)

### CODE REVIEW

File: aiogremlin/process/graph_traversal.py [hunk 1]
```diff
@@ -22,25 +22,23 @@ def __aiter__(self):
         return object
 
     async def toList(self):
-        results = []
-        async for result in self:
-            results.append(result)
-        return results
+        """Return results as ``list``."""
+        return [result async for result in self]
```
Comment (davebshow) [inline aiogremlin/process/graph_traversal.py]: This async list comprehensions are sweet, but do they work with Python 3.5?

**Comment by davebshow** [comment]
Other than my 1 comment, this is LGTM. Please address comment and I will merge.

File: aiogremlin/process/graph_traversal.py (hunk 1 above)
Comment (jirikuncar) [reply aiogremlin/process/graph_traversal.py]: I hope so 😆  We will find out when https://github.com/davebshow/aiogremlin/issues/11 is done.

Comment (davebshow) [reply aiogremlin/process/graph_traversal.py]: Spoiler: They don't :japanese_ogre: 

They were an enhancement introduced in Python 3.6 with [PEP 530](https://www.python.org/dev/peps/pep-0530/)

**Pull request merged**